gradle/README.md

```diff
@@ -2,6 +2,17 @@
 
 This Cloud Build builder runs the Gradle build tool.
 
+You should consider instead using an [official `gradle`
+image](https://hub.docker.com/_/gradle/) and specifying the `gradle` entrypoint:
+```yaml
+steps:
+- name: gradle:6.0.1-jdk11
+  entrypoint: 'gradle'
+  args: ['build']
```

Comment:
And this should be `['gradle', 'build']` for `latest`. I see the entrypoint is null.

Reply:
I think as instructions to users it's fine? Matches the maven instructions, and safely overrides any potential entrypoint that they might set?

Reply:
Ah never mind, I didn't see `entrypoint` above.